source/Calamari.Tests/KubernetesFixtures/Terraform/Clusters/GKE/gke.tf

```diff
@@ -1,3 +1,13 @@
+# Get latest version
+data "google_container_engine_versions" "main" {
+  location = local.region
+  project       = "octopus-api-tester"
+
+  # Since this is just a string match, it's recommended that you append a . after minor versions 
+  # to ensure that prefixes such as 1.1 don't match versions like 1.12.5-gke.10 accidentally.

```

Comment:
Very useful comment here! 🥇

Reply:
_cough_ I didn't write that, that was a copy paste _cough_

but I'll take the 🥇 😉